src/liedle/index.js

```diff
@@ -156,38 +156,28 @@ function colourWord(word) {
 }
 
 function colourLetter(tile) {
-  let index = window.secretWord.indexOf(tile.innerHTML.toLowerCase());
-  switch (index) {
-    case -1:
-      tile.dataset.type = "wrong";
-      break;
-    case parseInt(tile.dataset.colIndex):
-      tile.dataset.type = "right";
-      break;
-    default:
-      tile.dataset.type = "right-letter";
-      break;
+  let testChar = tile.innerHTML.toLowerCase();
+  if (secretWord.charAt(tile.dataset.colIndex) === testChar) {
+    tile.dataset.type = "right";
+  } else if (secretWord.includes(testChar)) {
+    tile.dataset.type = "right-letter";
+  } else {
+    tile.dataset.type = "wrong";
   }
 }
 
 function colourLetterFalsely(tile) {
-  let type;
-  let index = window.secretWord.indexOf(tile.innerHTML.toLowerCase());
-  // set the type to one of the incorrect types
-  switch (index) {
-    case -1:
-      type = Math.random() < 0.5 ? "right" : "right-letter";
-      break;
-    case parseInt(tile.dataset.colIndex):
-      type = Math.random() < 0.5 ? "wrong" : "right-letter";
-      break;
-    default:
-      type = Math.random() < 0.5 ? "right" : "wrong";
-      break;
+  let testChar = tile.innerHTML.toLowerCase();
+  if (secretWord.charAt(tile.dataset.colIndex) === testChar) {
+    tile.dataset.type =  Math.random() < 0.5 ? "wrong" : "right-letter";
+  } else if (secretWord.includes(testChar)) {
+    tile.dataset.type = Math.random() < 0.5 ? "right" : "wrong";;
```

Comment:
Unnesscery ";" at the end of the line, it is trivial and you can merge without changing it. Just a suggestion :)